### PR TITLE
Stats: Add support to output stats into `statsd`

### PR DESCRIPTION
**Pull request by bisko**
This PR adds logging into StatsD, so we can monitor the health of Jetmon better.

This PR also adds StatsD image to the local docker instance, so we can monitor the stats locally if we need to.

First version includes basic stats that we'll extend once we make sure this approach will work for our objectives.

### Testing instructions:

* Apply PR
* `docker compose build`
* `docker compose up`
* Check `https://localhost:8088` for the default Graphite Browser UI. 
* Make sure you can see some metrics being logged in `stats -> com -> jetpack -> jetmon -> docker -> jetmon -> dev -> com` in the Tree View on the side.
* Set the interval to the last 10 minutes to see anything in the UI. (It's not the best :( )

### CODE REVIEW

File: docker/run-jetmon.sh [hunk 1]
```diff
@@ -19,4 +19,5 @@ if [ ! -f config/db-config.conf ]; then
 	sed -e "s/<MYSQLDB_USER>/${MYSQLDB_USER}/g" -e "s/<MYSQLDB_ROOT_PASSWORD>/${MYSQLDB_ROOT_PASSWORD}/g" -e "s/<MYSQLDB_PORT>/${MYSQLDB_DOCKER_PORT}/g" -e "s/<MYSQLDB_DATABASE>/${MYSQLDB_DATABASE}/g" config/db-config-sample.conf > config/db-config.conf
 fi
 
+npm install
```
Comment (darssen) [inline docker/run-jetmon.sh:22]: Do we need this?
We already have `npm install` inside **Dockerfile_jetmon**

**Comment by darssen** [comment]
Wanted to start checking a bit but I'm unable to start the **statsd** service without errors since seems all the config folders (the volumes for the **statsd**  docker service) are empty.

File: docker/run-jetmon.sh (hunk 1 above)
Comment (bisko) [reply docker/run-jetmon.sh:22]: Oh, missed that entry. I wanted to re-run the service without having to rebuild the image. Going to remove it, as it's not a big blocker for the PR and it's done rarely enough.

**Review by chrisbliss18 — APPROVED** [review]
My testing looks good with these changes. I removed some unused code and retested with everything working as expected.

**Pull request merged**